shell/common/engine_unittests.cc

```diff
@@ -330,7 +332,8 @@ TEST_F(EngineTest, SpawnResetsViewportMetrics) {
     const double kViewHeight = 1024;
     old_viewport_metrics.physical_width = kViewWidth;
     old_viewport_metrics.physical_height = kViewHeight;
-    mock_runtime_controller->SetViewportMetrics(old_viewport_metrics);
+    mock_runtime_controller->SetViewportMetrics(kDefaultViewId,
+                                                old_viewport_metrics);
```

Comment:
This test uses `PlatformData.viewport_metrics` to check the metrics, which appears to be a single-view assumption. Should we update `PlatformData` as part of this change? I'm fine with punting this until later, but I'd consider adding a TODO with a tracking issue.

Reply:
Yeah I added a TODO at `RuntimeController::FlushRuntimeStateToIsolate`. I can add one more here.

Reply:
Actually, I don't think an extra one is needed here. `PlatformData.viewport_metrics` is definitely something we need to change in the future, which will for sure affect this test. The `TODO` I mentioned above should be enough.

Reply:
Could you put a TODO on `PlatformData::viewport_metrics` as well? Skipping the TODO here sounds good though 👍